### PR TITLE
Add ProductHunt.app (1.0)

### PR DESCRIPTION
**Pull request by thii**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download producthunt` is error-free.
- [x] `brew cask style --fix producthunt` left no offenses.
- [x] `brew cask install producthunt` worked successfully.
- [x] `brew cask uninstall producthunt` worked successfully.

### CODE REVIEW

File: Casks/producthunt.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'producthunt' do
```
Comment (vitorgalvao) [inline Casks/producthunt.rb]: The token reference was not followed. This needs to be `product-hunt`.

File: Casks/producthunt.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'https://s3.producthunt.com/mac/ProductHunt.dmg'
+  name 'Product Hunt'
+  homepage 'https://www.producthunt.com'
```
Comment (vitorgalvao) [inline Casks/producthunt.rb]: Isn’t there a specific page for the app?

**Comment by thii** [comment]
Fixed as comments. I thought no dash would make more sense since it matches their domain name.

**Comment by vitorgalvao** [comment]
> I thought no dash would make more sense since it matches their domain name.

The token reference is very explicit why opinions can’t be taken into account and instead we have strict rules. It’s the way to prevent duplicates, which used to be a huge issue.

**Pull request merged**